### PR TITLE
Removed deprecated functions

### PR DESCRIPTION
**Pull request by lakshya8066**
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did

**The method `getInstance()` from the type `Jenkins` is deprecated, we can instead use `get()` method as described in the documentation.
The method `toString()` from the type `FilePath` is also deprecated.
This PR updates the files accordingly.**

### CODE REVIEW

File: src/main/java/io/jenkins/plugins/intotorecorder/InTotoRecorder.java [hunk 1]
```diff
@@ -149,7 +149,7 @@ public boolean prebuild(AbstractBuild<?,?> build, BuildListener listener)  {
         this.cwd = build.getWorkspace();
         String  cwdStr;
         if (this.cwd != null) {
-            cwdStr = this.cwd.toString();
+            cwdStr = this.cwd+"";
```
Comment (adityasaky) [inline src/main/java/io/jenkins/plugins/intotorecorder/InTotoRecorder.java:152]: Is this the canonical way to make this type casting? I don't see a replacement method in Hudson, so I suppose it's okay.

**Review by adityasaky — APPROVED** [review]
LGTM, apart from minor nit. Thanks!

File: src/main/java/io/jenkins/plugins/intotorecorder/InTotoRecorder.java (hunk 1 above)
Comment (SantiagoTorres) [reply src/main/java/io/jenkins/plugins/intotorecorder/InTotoRecorder.java:152]: well, this is not exactly casting as far as I'm aware. I'm surprised they deprecated toString(). can we verify this returns the actual path and not some wonky retval like "FilePath@1234134" ?

Comment (adityasaky) [reply src/main/java/io/jenkins/plugins/intotorecorder/InTotoRecorder.java:155]: @lakshya8066 bump! I think we should get this merged.

**Pull request merged**